Week2/Homework/assets/question-six.js

```diff
@@ -0,0 +1,10 @@
+let nlCities = [];
+console.log('the value of the array is 0');
```

Comment:
are you sure that the value is '0'?

Reply:
It will be undefined

let nlCities = [];
console.log('the value of the array is undefined');
console.log(nlCities);

Reply:
Almost, it won't be undefined, but you are close, it cannot be undefined because you did give it a value, but what is the value?